account_commission/models/account_move.py

```diff
@@ -121,6 +121,13 @@ def unlink(self):
         ).write({"state": "settled"})
         return super().unlink()
 
+    def contact_by_agent_ids(self):
+        """
+        Returns a comma-separated string of agent IDs associated with the invoice.
```

Comment:
We use imperative style in the OCA (no s: return)

Reply:
fixed!